app/views/components/_datetime_fields.html.erb

```diff
@@ -1,6 +1,7 @@
 <%
  prefix = prefix
  field_name = field_name
+ date_only = date_only || false
```

Comment:
I think we should memoise this since it's used multiple times

date_only ||= date_only || false

Reply:
👍

Reply:
sorry it should be date_only ||= false